demos/text-field.html

```diff
@@ -80,8 +80,8 @@ <h2>Full Functionality JS Component (Floating Label, Validation)</h2>
             <div class="mdc-line-ripple"></div>
           </div>
           <p id="my-text-field-helper-text" class="mdc-text-field-helper-text"
-             aria-hidden="true" style="display:none;">
-            Helper Text (possibly validation message)
+             aria-hidden="true" style="display:none;"
+             >Helper Text (possibly validation message)
```

Comment:
Why is the helper text sensitive to leading whitespace now? AFAICT it was not previously?

If this is a limitation of the new baseline styles/mixin, I'd like to see us work it out if at all possible...